modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -204,6 +204,16 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 
 	// set up upstreams
 	for _, upstream := range h.Upstreams {
+		addr, err := caddy.ParseNetworkAddress(upstream.Dial)
+		if err != nil {
+			return err
+		}
+		if addr.PortRangeSize() != 1 {
+			h.logger.Error("multiple addresses (upstream must map to only one address)",
+				zap.String("address", upstream.Dial),
+			)
```

Comment:
Since we're returning the error with the same content, we don't need to log it here; it will get bubbled up and emitted that way.

Reply:
Done